src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -49,6 +51,7 @@
 from system import minijail
 from system import new_process
 from system import shell
+from zipfile import ZipFile
```

Comment:
Nit: instead of this, "from system import archive" and use archive.unpack instead. It's fine to pass trusted=True given that the file is checked in.

Reply:
Done